sechub-webui/src/main/resources/static/css/main.css

```diff
@@ -0,0 +1,3 @@
+body {
```

Comment:
Is there any chance that we (without much effort) could have some minimal css parts available (and classes/ids inside templates) , so the template output at browser is a little bit more readable from the beginning?